src/sub/tests/conftest.py

```diff
@@ -27,9 +31,10 @@
 
 logger = get_logger()
 
-ddb_process = None
 THIS_PATH = os.path.join(os.path.realpath(os.path.dirname(__file__)))
 UID = str(uuid.uuid4())
+IMAGE = "vitarn/dynalite:latest"
```

Comment:
make me think it should be parameterized; perhaps this as a default value in cfg.py

Reply:
I would opt to keep it here, extract a pytest fixture soon that is parallelizable that can have this in one place.  This would allow us to keep the configuration's surface area to a minimum for future maintainers.  If all are good with this approach, I have a todo item to do exactly this in my free-time at some point.

Reply:
Not let it be a todo item, but rather add it as an item to the backlog in Jira

Reply:
https://jira.mozilla.com/browse/COPS-790